src/vma/util/sys_vars.cpp

```diff
@@ -440,6 +440,27 @@ const char* mce_sys_var::cpuid_hv_vendor()
 #endif
 }
 
+void mce_sys_var::read_hv()
+{
+	char hyper_vendor_id[13] = {0};
+
+	if (cpuid_hv_vendor()) {
+		memcpy(hyper_vendor_id, cpuid_hv_vendor(), sizeof(hyper_vendor_id));
+
```

Comment:
Why call cpuid_hv_vendor() twice ? cpuid_hv_vendor should do the memcopy inside.:
int cpuid_hv_vendor(char* hyper_vendor_id) - return 0 on success and copy info to char*
in case we are not using x86 we will enter this if statement ("n/a") - please fix.

Reply:
done